app/backend/src/clj/teet/file/file_import.clj

```diff
@@ -27,14 +27,17 @@
 ;; Check for the file-suffix to match vektorio config values and the upload the file to vektorio
 (defmethod import-by-suffix :default
   [{:keys [vektorio-config conn s3] :as ctx}]
+  (log/info "File import by suffix triggered from aws S3 for file: " (:file-key s3))
   (let [suffix (uploaded-file-suffix ctx)]
     (when-let [extensions (and vektorio-config (get-in vektorio-config [:config :file-extensions]))]
       (when (extensions suffix)
         (let [db (d/db conn)
               file-id (ffirst (d/q '[:find ?file
                                      :in $ ?s3-key
-                                     :where [?file :file/s3-key ?s3-key]]
-                                   (:file-key s3)))]
+                                     :where
+                                     [_ :task/files ?file]  ;; Check that file belongs to task so we don't upload attachments
```

Comment:
this is now redundant with the is-task-file? check below

Reply:
The is-task-file? seems to be doing a lot of stuff compared to just having the where clause, but you're correct I didn't notice that check was already added added.